### ISSUE TITLE
`ReactPerf.getMeasurementsSummaryMap` deprecation warning on React 15.3

### ISSUE THREAD

**Issue by bebraw** [RamonGebben/react-perf-tool#13]
I'm getting the following warning:

```
Warning: `ReactPerf.getMeasurementsSummaryMap(...)` is deprecated. Use `ReactPerf.getWasted(...)` instead.
```

Also seeing an error after that:

```
Uncaught TypeError: Cannot read property 'split' of undefined
```

If you want to study further, I set up [a branch](https://github.com/reactabular/reactabular/tree/perf-tool) showcasing the error. `npm install`, `node_modules/.bin/lerna bootstrap`, `npm start` to get it running.

![screenshot 2016-08-04 20 00 34](https://cloud.githubusercontent.com/assets/166921/17410846/26d3494c-5a7e-11e6-9d0d-210b6ba053c1.png)

**Comment by joyfulelement** [comment]
+1 
Have the exact same issue with the library line up of:
- `react-addons-perf`: `15.3.0`
- `react-perf-tool`: `0.1.5`
- `react`: `15.3.0`

**Comment by RamonGebben** [comment]
To let you guys know I'm working on resolving this issue.
The `Warning: `ReactPerf.getMeasurementsSummaryMap(...)` is deprecated. Use `ReactPerf.getWasted(...)` instead.` was being generated by the `console.log` I left laying around so that's already resolved. 
How ever since react-addons-performance has updated their format I would need to dive into the new format.

Hopefully have an update ready by the end of the week.

**Comment by markmur** [comment]
Any updates on this?

**Comment by tbranyen** [comment]
@markmur I fixed this in https://github.com/RamonGebben/react-perf-tool/pull/14

**Comment by RamonGebben** [comment]
Just published new version(0.1.7) including @tbranyen fix to npm.
Thanks for solving this.
